pkg/values/values.go

```diff
@@ -65,6 +65,7 @@ const (
 	Rpi3    Model = "rpi3"
 	Rpi4    Model = "rpi4"
 	AgxOrin Model = "agx-orin"
+	OrinNX  Model = "orin-nx"
```

Comment:
For consistency I'd say NxOrin, but maybe it makes more sense OrinNx, but if that's the case maybe we can also rename the AgxOrin one to OrinAgx wdyt @mudler @Itxaka

Reply:
cant see the image but Im wondering...do they call them agx-orin and orin-nx????

Can we call them agx-orin and agx-orin-nx or will that also not reflect the proper names @jordankrp ?

Ughhhh namingg